### DISCUSSION TITLE
Conda package 'moose-dev' is currently at version '2024.06.17' and the required version is '2024.05.13'.

### DISCUSSION THREAD

**Discussion by puraw** [neams-th-coe/cardinal#924] [General]
CMake Error at /home/monolith/miniconda3/envs/moose/share/cmake-3.25/Modules/CheckFunctionExists.cmake:91 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
  src/io/CMakeLists.txt:95 (check_function_exists)

**Comment by aprilnovak** [comment]
Hi @puraw, I'm assuming the error you are getting is the one listed in the title? If so, can you please try upgrading your MOOSE conda package?

```
conda activate base
conda env remove -n moose
conda create -n moose moose-dev=2024.07.03
conda activate moose
```
